### DISCUSSION TITLE
Student Verification Screen Not Proceeding

### DISCUSSION THREAD

**Discussion by omicreativedev** [community/community#129139] [GitHub Education]
### Hi everyone,

I am reverifying my student account and after adding my new email address from the university I transferred to, the system said it needed more information and for me to upload my ID or documentation. I did that, but upon clicking the button to proceed, the screen reloads and nothing happens. I tried to upload and submit several times but the screen just returns to the upload screen where I must upload again. I'm not getting any errors. I tried it in several browsers. And I cannot find a contact email for github to resolve it. I've also tried many things posted in the forums. It's just not proceeding to verify with the documents I upload. The documents are the right size, format, etc.

**Comment by davevad93** [comment]
Hi @omicreativedev ,

Check out this discussion #126276 , this user solved it using the webcam instead of uploading the documentation.

If you need to contact support you can go [here](https://support.github.com/contact/education) but GitHub Education support no longer provides individual manual reviews and redirects here in the Community discussions, so you could try opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.
